<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Node View Dummy</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(70,70,70);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_f1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>726</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Filter 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_f2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>726</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Filter 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_f3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>726</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>Filter 3</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>821</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 50, 50);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>1181</width>
       <height>701</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(80, 80, 80);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>722</y>
      <width>1181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(70, 70, 70);</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>widget</zorder>
   <zorder>pushButton_f1</zorder>
   <zorder>pushButton_f2</zorder>
   <zorder>pushButton_f3</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
